docs/_ug/commandSummary/itemCommands/find.md

```diff
@@ -1,2 +1,2 @@
 <!-- markdownlint-disable-file first-line-h1 -->
-<b>find n/ITEM_NAME</b><br><br><u>Valid Example:</u><br>find n/Potato
+find ITEM_NAME
```

Comment:
I believe that we the syntax should be 

find KEYWORD [KEYWORDS]...

as we do not need the full item name and can have multiple keywords

Reply:
oh i see! i did not know we can have multiple keywords, will change!